Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DPEComponentAdapter.cpp

```diff
@@ -39,11 +39,16 @@ namespace AZ::DocumentPropertyEditor
         if (m_componentId == componentId)
         {
             m_queuedRefreshLevel = AzToolsFramework::PropertyModificationRefreshLevel::Refresh_Values;
+            QPointer<QObject> stillAlive(&m_stillAlive);
```

Comment:
So abuse of [QPointer guarded pointer](https://doc.qt.io/qt-6/qpointer.html#details) is being used to detect if the instance of this ComponentAdapter has has been deleted.
I guess that works